ecs-cli/modules/commands/flags.go

```diff
@@ -48,6 +48,7 @@ const (
 	SubnetIdsFlag                   = "subnets"
 	VpcIdFlag                       = "vpc"
 	InstanceTypeFlag                = "instance-type"
+	InstanceRoleFlag                = "role"
```

Comment:
Can you name this as `instance-role`, or `instance-profile-role`? Helps disambiguate this from other task/service roles that may need to be specified elsewhere.